x/fundraising/types/bid.go

```diff
@@ -12,10 +12,18 @@ func (b Bid) GetBidder() sdk.AccAddress {
 	return addr
 }
 
-func (b *Bid) SetWinner(status bool) {
-	b.IsWinner = status
+func (b *Bid) SetMatched(status bool) {
+	b.IsMatched = status
 }
 
-func (b Bid) GetExchangedSellingAmount() sdk.Int {
-	return b.Coin.Amount.ToDec().QuoTruncate(b.Price).TruncateInt()
+func (b Bid) GetBidSellingAmount(denom string) sdk.Int {
+	var bidSellingAmt sdk.Int
+
+	if b.Coin.Denom == denom {
+		bidSellingAmt = b.Coin.Amount.ToDec().QuoTruncate(b.Price).TruncateInt()
+	} else {
+		bidSellingAmt = b.Coin.Amount
+	}
+
+	return bidSellingAmt
```

Comment:
Is there use case when this function received the selling coin denom? If not, let's update the `denom` to `payingCoinDenom` to improve readability.
```suggestion
func (b Bid) GetBidSellingAmount(payingCoinDenom string) sdk.Int {
	var bidSellingAmt sdk.Int

	if b.Coin.Denom == payingCoinDenom {
		bidSellingAmt = b.Coin.Amount.ToDec().QuoTruncate(b.Price).TruncateInt()
	} else {
		bidSellingAmt = b.Coin.Amount
	}

	return bidSellingAmt
```

Reply:
I changed as suggested.